webpack.base.js

```diff
@@ -64,4 +54,15 @@ module.exports = ({ entry, mode, filename, types }) => ({
       }),
     ],
   },
+
+  plugins:
+    // do not replace in unit test in order to test behaviors related to build env variables
+    mode !== 'development'
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This breaks `yarn dev`, since it is also using a development mode

Reply:
Nice catch! 👀